contracts/interfaces/IAxelarGMPTransfer.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+/**
+ * @title IAxelarGMPTransfer
+ * @dev Interface for tracking asset value transfers using General Message Passing (GMP) calls in the Axelar Network.
+ * This interface defines an event that should be emitted when a GMP transfer occurs,
+ * allowing for standardized volume tracking across different implementations.
+ */
+interface IAxelarGMPTransfer {
+    /**
+     * @dev Emitted when a GMP transfer occurs, providing details for volume tracking.
+     * @param sender The address of the caller that initiated the transfer on the source chain.
+     * @param sourceChain The Axelar chain identifier of the source chain.
+     * @param destinationChain The Axelar chain identifier of the destination chain.
+     * @param destinationAddress The address of the contract on the destination chain that receives the transfer.
+     * @param recipientAddress The address of the final recipient of the transferred assets on the destination chain.
+     * @param tokenAddress The address of the token contract on the source chain.
+     * @param amount The amount (in atomic units) of tokens transferred.
+     * @param decimals The number of decimal places for the token.
+     */
+    event AxelarGMPTransfer(
```

Comment:
The same event shouldn't be used for both source and destination.

On the source side, `sourceChain` and `sender` are obvious from the event. `sourceChain` identifier isn't actually available to apps even (it's an identifier at the protocol level, we could add it to the gateway but it'll still be costly to query it).

On the destination side, `destinationChain` isn't available similarly, and `destinationAddress` isn't available as a string (costly to convert) but also not needed, these values are clear from the event itself. `string sourceAddress` should be used to mimic GMP. `bytes sender` could be added as well for the true sender in case it's different than the source contract address (e.g. ITS transfer)

Reply:
While I'd prefer to require the event to only be emitted on the destination chain, I suppose it's convenient to allow source to emit it as well, if the destination isn't a contract it controls, or if the GMP call wasn't executed/takes a while due to finality.

Reply:
I was wondering if we should have different events depending on whether it was being emitted from the source chain or the executable on the destination chain. So yeah, we can consider creating two different events, one for src, one for dest?

Reply:
Yep, should have different events with only the applicable fields in each case